Adding Traslathon explanation (#4)
ropenscilabs/FTCFOSS-Guide: translathon.qmd

title: "Mini-translathon"
---

## Overview

Translathons are 2-hour events designed as a combination of *translation* with the ways and energy of a *hackathon* (a dedicated period of time where participants make contributions to selected FOSS projects).
Translathons are live and synchronous: participants contribute, learn, and connect with others.

> 

Author: ```suggestion
Plan a community call to provide guidance on how to get started with translation contributions and different translations and localization efforts that exist for a human language in the community.
```

> The community call provided guidance on how to get started with translation contributions.

Author: Plan a community call to provide guidance on how to get started with translation contributions and different translations and localization efforts that exist for a human language in the community.

title: "Mini-translathon"
---

## Overview

> Translathons are 2-hour events designed as a combination of *translation* (the communication of the meaning of a source-language text by means of an equivalent target-language text) with the ways and energy of a *hackathon* (a dedicated period of time where participants make contributions to selected FOSS projects).

Reviewer: I don't think we need to define translation

> Translathons are 2-hour events designed as a combination of *translation* (the communication of the meaning of a source-language text by means of an equivalent target-language text) with the ways and energy of a *hackathon* (a dedicated period of time where participants make contributions to selected FOSS projects).

Author: ```suggestion
Translathons are 2-hour events designed as a combination of *translation* with the ways and energy of a *hackathon* (a dedicated period of time where participants make contributions to selected FOSS projects).
```

Translathons are live and synchronous: participants contribute, learn, and connect with others.

rOpenSci staff prepare the event by identifying materials in need of translation or review, creating and labeling pull requests for participants to work on, and setting up a GitHub Project board to track progress.
During the event, after a brief introduction on how the event is structured and how the translation process works, participants select a pull request to work on.
Mentors and editors provide real-time support in breakout rooms or the main room.
All contributors are also added to the multilingual channel in the rOpenSci Slack, where they can continue discussing and refining translations after the event.

During and after the event, participants are invited to complete an optional feedback survey collecting insights on their experiences, challenges faced, and suggestions for improvement.

> The community call provided guidance on how to get started with translation contributions and different translations and localization efforts that exist for a human language in the community.

Author: ```suggestion

```

## Mentors and Editors

> The main goal is translathon events is to ensure participants have a good experience:

Reviewer: ```suggestion
The main goal of translathon events is to ensure participants have a good experience:
```

    They help participants get started, troubleshoot issues, and guide them through the workflow.

-   **Editors:** experienced translators responsible for reviewing and maintaining the quality and consistency of translations.
    Editors check terminology, style, and adherence to the community’s translation agreements.

Mentors and Editors sign up to participate beforehand and review the [translation process and rules](https://translationguide.ropensci.org), and the translation that need contribution.
They are preferably people who have already contributed, as reviewers or editors, to rOpenSci's translations and know how the tools used in the process work and are also aware of the translation agreements of the language in which they are involved.

Because preparing and reviewing translations requires additional effort, we suggest offering mentors and editors an honorarium to recognize their time and contributions.

> In our 2024 pilot, the translathon was hosted in Portuguese in conjunction with LatinR, where many members of the Latin American R community gathered.
> We focus the translation effort to one resource: rOpenSci Packages: Development, Maintenance, and Peer Review.

> > NumFocus provide the funding to pay mentors and editors a \$200 USD honorarium for participating in the translathons.

Reviewer: ```suggestion
> NumFocus provided the funding to pay mentors and editors a \$200 USD honorarium for participating in the translathon.
```

## Timeline

See [Events Overview](events-overview.html) for the full timeline of paired events.

> The timeline is setup for a single event, but repeat events are easy to insert after the first three tasks.

Reviewer: I know this is my wording, but I'll update the hackthons if you agree here 😅 
```suggestion
The timeline is setup for a single event, but repeat events are easy to insert.
```

Because preparing and reviewing translations requires additional effort, we suggest offering mentors and editors an honorarium to recognize their time and contributions.

> In our 2024 pilot, the traslaton was hosted in Portuguese in conjunction with LatinR, where many members of the Latin American R community gathered.
> We focus the translation effort to one resource: rOpenSci Packages: Development, Maintenance, and Peer Review.
> NumFocus provide the funding to pay mentors and editors a \$200 USD honorarium for participating in the traslathons.

## Timeline

See [Events Overview](events-overview.html) for the full timeline of paired events.

The timeline is setup for a single event, but repeat events are easy to insert after the first three tasks.

|  |  |  |
|-------------------------|-----------------------|-------------------------|
| **Weeks Before** | **Task** | **Platform** |
| 10 | Choose date for traslathon or an event to joint organization. |  |

> | 8 | Open call for mentors/editors at hands-on events | [Blog](https://ropensci.org/blog/) / Registration/ [Social](https://hachyderm.io/@rOpenSci) (include Slack workspaces) |

Reviewer: I added a template for social media posts, let's link to that and keep this simple.
```suggestion
| 8 | Open call for mentors/editors at hands-on events | [Blog](https://ropensci.org/blog/) / Registration/ [Social](templates_social.html) |
```

> | 4–6 | Open call for participants at traslaton(s). | [Blog](https://ropensci.org/blog/) / [Registration](https://www.eventbrite.cl/e/traducao-hackathon-traslaton-ropensci-tickets-1045507530167) (may vary if organized jointly with another event) / [Social](https://hachyderm.io/@rOpenSci) (include Slack workspaces) |

Reviewer: ```suggestion
| 4–6 | Open call for participants at translathon(s) | [Blog](https://ropensci.org/blog/) / [Registration](https://www.eventbrite.cl/e/traducao-hackathon-traslaton-ropensci-tickets-1045507530167) / [Social](templates_social.html)|
```

| 4–6 | Event Advertising | [Website](https://ropensci.org/events/) (may vary if organized jointly with another event, example [LatinR Website](https://latinr.org/pt/cronograma/translaton/translaton-pt.html)) / [Social](https://hachyderm.io/@rOpenSci) (include Slack workspaces) |

> | 1–2 | Send instructions to mentors/editors | Email (Process) |

Reviewer: Add email templates for the translathons? Or remove the "(Process)" part.
```suggestion
| 1–2 | Send instructions to mentors/editors | Email ([Process](templates_emails.html)) |
```

> | 0–1 | Send instructions to participants | Email (only if we request registration) |

Reviewer: ```suggestion
| 0–1 | (If participants register) Send instructions to participants | Email |
```

> | 0 | Run traslaton(s) | Zoom / [Docs](https://translationguide.ropensci.org/) / [Project Board](https://github.com/orgs/ropensci/projects/7?query=is%3Aopen+sort%3Aupdated-desc) |

Reviewer: ```suggestion
| 0 | Run translathon(s) | Zoom / [Docs](https://translationguide.ropensci.org/) / [Project Board](https://github.com/orgs/ropensci/projects/7?query=is%3Aopen+sort%3Aupdated-desc) |
```

> | After event | Evaluate participant feedback | Survey form (Airtable) |

Reviewer: Add form content to the templates_form.html and link here?
